tutorials/Numerics/TimeStepping/multirate_rk.jl

```diff
@@ -0,0 +1,134 @@
+#     \left(\frac{1}{\tilde{c}_i^O}\boldsymbol{r}_i + {\mathcal{T}_{f}}(\boldsymbol{w}_{i,j})\right),\\
+#     & \quad\quad i = 2, \cdots, s^O + 1 \text{ and } k = 2, \cdots, s^I + 1,\nonumber \\
+#     \boldsymbol{Q}_i &= \boldsymbol{w}_{i,s^I + 1}
+# \end{align}
+# ```
+#
+# where the tilde parameters denote increments per RK stage:
+#
+# ```math
+# \begin{align}
+#     \tilde{a}_{ij} &= \begin{cases}
+#         a_{i,j} - a_{i-1, j} & \text{if } i < s + 1 \\
+#         b_j - a_{s,j} & \text{if } i = s + 1
+#     \end{cases},\\
+#     \tilde{c}_{i} &= \begin{cases}
+#         c_{i} - c_{i-1} & \text{if } i < s + 1 \\
+#         1 - c_{s} & \text{if } i = s + 1
+#     \end{cases},
+# \end{align}
+# ```
+#
+# where the coefficients ``a``, ``b``, and ``c`` correspond to the Butcher
+# tableau for a given RK method. The superscripts O`` and I`` denote the
+# *outer* (slow) and *inner* (fast) components of the multirate method
+# respectively. Thus, tilde coefficients should be associated with the RK
+# method indicated by the superscripts. In other words, the RK methods
+# for the slow ``{\mathcal{T}_{s}}`` and fast
+# ``{\mathcal{T}_{f}}`` components have Butcher tables given by:
+#
+# ```math
+# \begin{align}
+#     \begin{array}{c|c}
+#     \boldsymbol{c}_{O} &\boldsymbol{A}_{O} \\
+#     \hline
+#     & \boldsymbol{b}_O^T
+#     \end{array}, \quad
+#     \begin{array}{c|c}
+#     \boldsymbol{c}_{I} &\boldsymbol{A}_{I} \\
+#     \hline
+#     & \boldsymbol{b}_I^T
+#     \end{array},
+# \end{align}
+# ```
+#
+# where ``\boldsymbol{A}_O = \lbrace a_{i,j}^O\rbrace``, ``\boldsymbol{b}_O = \lbrace b_i^O \rbrace``, and
+# ``c_O = \lbrace c_i^O \rbrace`` (similarly for ``\boldsymbol{A}_I``, ``\boldsymbol{b}_I``, and ``\boldsymbol{c}_I``).
+# The method described here is for an explicit RK outer method with ``s`` stages.
+# More details can be found in [Schlegel2012](@cite).
+
+# The acoustic wave test case used in this tutorial represents a global-scale
+# problem with inertia-gravity waves traveling around the entire planet.
+# It has a hydrostatically balanced initial state that is given a pressure
+# perturbation.
+# This initial pressure perturbation causes an acoustic wave to travel to
+# the antipode, coalesce, and return to the initial position. The exact solution
+# of this test case is simple in that the (linear) acoustic theory allows one
+# to verify the analytic speed of sound based on the thermodynamics variables.
+# The initial condition is defined as a hydrostatically balanced atmosphere
+# with background (reference) potential temperature.
+
+ode_solver = ClimateMachine.MultirateSolverType(
+    splitting_type = ClimateMachine.HEVISplitting(),
+    slow_method = LSRK54CarpenterKennedy,
+    fast_method = ARK2GiraldoKellyConstantinescu,
+    implicit_solver_adjustable = true,
+    timestep_ratio = 100,
+)
+
+timeend = FT(3600)
+C = FT(5)
+cfl_direction = HorizontalDirection()
+run_acousticwave(ode_solver, C, cfl_direction, timeend)
+
```

Comment:
It outputs a number after the simulation. It is better to explain what that stands for.

Reply:
Oh, definitely. Thank you. We actually had decided to suppress the output in the other tutorials This one must have slipped!